### PR TITLE
[FEAT] Town Sign

### PR DESCRIPTION
**Pull request by c-py**
# Description

Town Sign!

sign1
sign2
sign3

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

localhost

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -5252,4 +5252,14 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  "Town Sign": {
+    description: "Show your farm ID with pride!",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/1284.png",
```
Comment (elias7905) [inline metadata/metadata.ts:5259]: There isn't a image in the files to show for OS metadata. Not a huge deal but do you need any help with the image?

Comment (elias7905) [reply metadata/metadata.ts:5259]: I created a quick image for the town sign. Maybe you could include it in.

Comment (c-py) [reply metadata/metadata.ts:5259]: @elias7905 Good catch, forgot to generate this!!

**Comment by elias7905** [comment]
Minor suggestion (not sure if it's already implemented): have the town sign placed down by default when a new user mints their farm.

**Pull request merged**